backend/app/api/v1/music_api.py

```
@@ -0,0 +1,20 @@
+router = APIRouter()
+
+
+class PromptRequest(BaseModel):
+    prompt: str
+
+
+@router.post("/users/{user_key}/music")
+async def generate_music(
+        user_key: str,
+        request: PromptRequest,
+        db=Depends(get_mongo_client)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)